PowerEditor/src/ScitillaComponent/Buffer.cpp

```diff
@@ -1381,6 +1381,7 @@ bool FileManager::loadFileData(Document doc, const TCHAR * filename, char* data,
 			{
 				if (fileFormat._encoding == SC_CP_UTF8)
 				{
+					_pscratchTilla->execute(SCI_SETCODEPAGE, SC_CP_UTF8);

```

Comment:
@eljefe7000 Seems this is a unrelated change and shouldn't be part of this PR.

Reply:
@eljefe7000  yes, @chcg  is right. Why this line?